testsuite/tests/DnsServerAPI.rb

```diff
@@ -84,32 +84,6 @@ def main
           }
         },
         "sysconfig" => {
-          "SuSEfirewall2"     => {
```

Comment:
I guess it is an unrelated clean-up, right?

Reply:
Unrelated, yes, but as SuSEfirewall2 is now unsupported this mockup is invalid or not necessary.